fetch-utils.js

```diff
@@ -27,3 +27,34 @@ export async function signOutUser() {
 }
 
 /* Data functions */
+
+
+export async function createItem(item) {
+    return await client.from('shopping_list').insert(item);
+
+}
+
+export function renderItem(item) {
+    const li = document.createElement('li');
+    li.textContent = item.quantity + ' ' + item.item;
+    return li;
+}
```

Comment:
technically this is not a fetch function, it'd belong better in a file called "render-utils.js" or if you've just got the one "app.js" would be okay